drivers/lambda: Add build support for vboxguest and vboxsf drivers

From: Rajiv M Ranganath <rajiv.ranganath@atihita.com>


---
 drivers/lambda/Kconfig  |    7 +++++++
 drivers/lambda/Makefile |    2 ++
 2 files changed, 9 insertions(+)

diff --git a/drivers/lambda/Kconfig b/drivers/lambda/Kconfig
index cb1fdb3..431402c 100644
--- a/drivers/lambda/Kconfig
+++ b/drivers/lambda/Kconfig
@@ -8,4 +8,11 @@ config LAMBDA_DRIVER_UPDATES
 
 if LAMBDA_DRIVER_UPDATES
 
+config LAMBDA_VBOX
+        tristate "Add support for vbox guest drivers"
+        depends on PCI
+        ---help---
+          Add support for vboxguest and vboxsf drivers. We do not integrate
+          vboxvideo driver into the tree.
+
 endif # LAMBDA_DRIVER_UPDATES
diff --git a/drivers/lambda/Makefile b/drivers/lambda/Makefile
index 49ca816..e55ddcb 100644
--- a/drivers/lambda/Makefile
+++ b/drivers/lambda/Makefile
@@ -1,3 +1,5 @@
 # Lambda Linux Driver Updates
 
+obj-$(CONFIG_LAMBDA_VBOX)   += vboxguest/ vboxsf/
+
 ########################
